consumer/src/main/kotlin/streams/kafka/KafkaSinkConfiguration.kt

```diff
@@ -71,7 +77,14 @@ data class KafkaSinkConfiguration(val zookeeperConnect: String = "localhost:2181
             )
         }
 
-        private fun validate(config: Map<String, String>) {
+        private fun validate(config: KafkaSinkConfiguration) {
+            validateConnection(config.zookeeperConnect, "zookeeper.connect", false)
+            validateConnection(config.bootstrapServers, CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG)
```

Comment:
do all need to be reachable or just one?

Reply:
Kafka uses Zookeeper so we only need that the Kafka broker is reachable